src/workspaces/list/WorkspacesListModals.ts

```diff
@@ -29,7 +29,10 @@ export const WorkspacesListModals = (props: WorkspacesListModalsProps): ReactNod
       h(NewWorkspaceModal, {
         cloneWorkspace: userActions.cloningWorkspace,
         onDismiss: () => setUserActions({ cloningWorkspace: undefined }),
-        onSuccess: ({ namespace, name }) => goToPath('workspace-dashboard', { namespace, name }),
+        onSuccess: () => {
+          refreshWorkspaces();
```

Comment:
And ack… we don't want to change the behavior for GCP workspaces, which will remain synchronous (even when going through the v2 API). 

I think this warrants unit test coverage for the difference in behavior.

Reply:
A couple more things--

When I cloned from the menu on the workspace dashboard, I DID get brought to the new workspace. So currently there is a difference in behavior depending on the location (cloning from the workspace list left me on the workspace list).

We will have to update the splash screen text that is shown while the clone is in progress. 

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/3eb78b9f-2b1d-4b75-acec-97541feb7b3d)

Reply:
> Or you could maintain the old behavior here until the cut-over to the v2 API.
I could put back the `goToPath('workspace-dashboard', { namespace, name }` in the cloning portion of `WorkspacesListModals`

> And ack… we don't want to change the behavior for GCP workspaces, which will remain synchronous (even when going through the v2 API).

We should probably talk about that more broadly. So long as they are sync, they will just be in a ready state immediately, and be available. So maybe the consisten experience is preferable?

> When I cloned from the menu on the workspace dashboard, I DID get brought to the new workspace.
That's because I haven't made any updates to the dashboard at all. That is in [WOR-718](https://broadworkbench.atlassian.net/browse/WOR-718)

> We will have to update the splash screen text that is shown while the clone is in progress.
I just noticed that myself... I feel like the clone behavior is diverging enough from the create, that those need to be split (we could probably split out the common components).

Reply:
For the purposes of this PR, I think it would help limit the scope if you did not change the pre-existing behavior (since it is still fully synchronous).

When we do change the behavior to call the v2 endpoint, I think that tests do need to be added to cover the behavior in both places. And the splash screen will need to be updated.

I agree that we should talk in standup about whether we want the GCP clone experience to be the same as it used to be.